content_handler/runtime_holder.cc

```diff
@@ -269,6 +273,8 @@ void RuntimeHolder::CreateView(
 
   runtime_->dart_controller()->dart_state()->SetReturnCodeCallback(
       [this](int32_t return_code) { return_code_ = return_code; });
+
+  runtime_->SetSemanticsEnabled(true);
```

Comment:
Are we sure we want to do this unconditionally?  Enabling semantics will have an impact on performance.

Reply:
I've removed this until we have more data.